python/nav/bin/navcheckservice.py

```diff
@@ -31,7 +31,7 @@
 
 from nav.logs import init_stderr_logging
 from nav.statemon import checkermap
-import nav.statemon.checker  # necessary for relative imports on python3
+import nav.statemon.checker  # noqa: F401 - necessary for relative imports
```

Comment:
This import was added by me, 7 years ago.  I don't understand the code comment any more, but the commit comment read as follows:

> also add a seemingly useless import for plugin imports to work under py3

I can no longer understand or remember why this import was necessary at the time, but the comment seems to indicate that this was necessary for `navcheckservice` to properly detect and load all the service checker plugins.  

Currently, though, it seems `navcheckservice` still works if the import is removed.
I think the primary verification that this still works is trying to run `navcheckservice` and give an invalid name for a servicemon checker plugin:

```console
$ navcheckservice  127.0.0.1 beep boop
usage: navcheckservice [-h] IP SYSNAME HANDLER
navcheckservice: error: argument HANDLER: invalid choice: 'boop' (choose from 'dhcp', 'http', 'smtp', 'imap', 'ftp', 'dns', 'pop3', 'https', 'radius', 'dummy', 'mysql', 'ssh', 'oracle', 'postgresql', 'rpc', 'ldap', 'dc', 'port', 'imaps', 'smb')
$
```

The response lists all the valid checker names, which indicates that they were all in fact discovered.